db/db_impl/db_impl.cc

```diff
@@ -4964,6 +4964,11 @@ Status DBImpl::VerifyChecksumInternal(const ReadOptions& read_options,
     sv_list.push_back(cfd->GetReferencedSuperVersion(this));
   }
 
+  // `bytes_read` stat is enabled based on compile-time support and cannot
+  // be dynamically toggled. So we do not need to worry about `PerfLevel`
+  // here, unlike many other `IOStatsContext` / `PerfContext` stats.
+  IOSTATS_RESET(bytes_read);
```

Comment:
Just realized resetting is a bad approach because a higher layer might also be tracking `bytes_read`. I will change it to not reset and instead increase `VERIFY_CHECKSUM_READ_BYTES` by the delta of `bytes_read`.

Reply:
Done.